app/controllers/admin/communities_controller.rb

```diff
@@ -221,4 +222,8 @@ def payment_settings_api
     TransactionService::API::Api.settings
   end
 
+  def escrow_payments?(community)
+    MarketplaceService::Community::Query.payment_type(community) == :braintree
```

Comment:
Well, these find_by_id, and I think find as well (which is the underlying implementation of payment_type) default to either using passed parameter as id or call .id on passed object.